### DISCUSSION TITLE
Join Multiple Presence Channel

### DISCUSSION THREAD

**Discussion by kishanhitk** [supabase/supabase#12387] [Questions]
Tl;dr: Can I join multiple presence channels in a single component simultaneously?

Eg: If a user is online in Central Par then they should join the Central Park channel, New York channel, and USA channel and should be shown active on all the channels.

Right now I tried joining multiple channels at once, but only the channels I join first, show me in the state, other channels show an empty state. Say, If I move the function to join the New York channel, then I am shown active in New York, but no info on other channels. And if I join the USA channel first, then I am shown active in New York, but no info on other channels.
